clouddriver-consul/src/main/groovy/com/netflix/spinnaker/clouddriver/consul/api/v1/ConsulAgent.groovy

```diff
@@ -19,11 +19,6 @@ package com.netflix.spinnaker.clouddriver.consul.api.v1
 import com.netflix.spinnaker.clouddriver.consul.api.v1.services.AgentApi
 import com.netflix.spinnaker.clouddriver.consul.config.ConsulConfig
 import com.netflix.spinnaker.clouddriver.consul.config.ConsulProperties
-import com.squareup.okhttp.OkHttpClient
-import retrofit.RestAdapter
-import retrofit.client.OkClient
-
-import java.util.concurrent.TimeUnit
```

Comment:
If there are some unrelated refactors, please separate them into their own commits to make this easier to review.

Reply:
they are not unrelated. I removed unused retrofit imports.

Reply:
Side note -- please don't resolve conversations that you didn't initiate.

To me these are unrelated.  As in, they don't relate the commit message.  And I'm sticking to my story that smaller commits are easier to review.

Reply:
I added back the imports